src/database/pipelines/repopulateRawContactsAndVotes.ts

```diff
@@ -147,3 +143,6 @@ async function* asBatches<T>(
   }
   if (batch.length > 0) yield batch;
 }
+
+await RepopulateRawContactsAndVotes.run(createDB());
+process.exit(0);
```

Comment:
i've been putting the actual invocations of different scripts in `src/scripts/...`, so they can all be run as `npm run tsx src/scripts/...`. i'm not sure why i've been doing this, but it'd be nice to be consistent!

Reply:
Oh dang, missed that, will swap